src/components/CharacterEventQueue.cpp

```diff
@@ -305,7 +305,7 @@ namespace REGoth
         break;
 
       case AI::StateMessage::ST_Wait:
-        message.waitTime -= bs::gTime().getFixedFrameDelta();
+        message.waitTime -= bs::gTime().getFrameDelta();
```

Comment:
If you're running with VSync, the dynamic one will still be capped to 60 fps. Since my laptop struggles to reach 60 fps (integrated graphics), not using fixed-update increases the frame-rate by a lot.